bittytax/conv/parsers/binance.py

```diff
@@ -328,8 +331,8 @@ def get_buy_quantity(op_rows):
 
 DataParser(DataParser.TYPE_EXCHANGE,
            "Binance Deposits/Withdrawals",
-           ['Date(UTC)', 'Coin', 'Amount', 'TransactionFee', 'Address', 'TXID', 'SourceAddress',
-            'PaymentID', 'Status'],
+           ['Date(UTC)', 'Coin', 'Network', 'Amount', 'TransactionFee', 'Address', 'TXID',
```

Comment:
Not sure why you added this header again? It's already defined above https://github.com/BittyTax/BittyTax/blob/02a6f18cb3489b0228f68494c326f4ac5748862e/bittytax/conv/parsers/binance.py#L324